### ISSUE TITLE
[Accessibility] [Screen reader -Intro]: Name is not properly defined for ‘Intro’. [Sev3][WCAG4.1.2]

### ISSUE THREAD

**Issue by Laxmidurga** [microsoft/tsdoc.org-website#31]
**Test Environment:**
OS Build: Windows 11
Version: 21H2 (OS Build 22000.318)
Browser: Edge
Version 96.0.1054.43 (Official build) (64-bit)
URL: https://tsdoc.org/
Screen reader: Narrator

**Repro Steps:**
1.	Open the above URL and login with valid credentials.
2.	TSDoc page will get displayed.
3.	Turn on screen reader and navigate to “Intro” link.
4.	Now, check whether the name is properly defined or not. 

**Actual:**
Name is not properly defined for ‘Intro’ link. Name is defined as ‘Link Intro1 of 5’ and the role is defined as ‘link’
So, screen reader is narrating link twice as ‘link link intro 1 of 5’ which is not appropriate. Because first it is narrating role as link and later name as link intro 1 of 5.

**Expected:**
Name should be properly defined for ‘Intro’ link. Name should be defined as ‘Intro 1 of 5’ and the role is defined as ‘link’
So, screen reader Should narrate link ‘Link intro 1 of 5’.

**User Impact:**
When interactive elements have an incorrect name, it is difficult for people using assistive technology (like screen readers) to understand the purpose of control and its relationship to other content.

**Note:**
Issue is observed for Tags, Help, GitHub, Playground links.

**WCAG Reference:**
[Understanding Success Criterion 4.1.2: Name, Role, Value (w3.org)](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value)

https://user-images.githubusercontent.com/88436727/150644071-055b510e-cc63-4e8a-b104-775154387020.mp4
